### ISSUE TITLE
Embed jitsi in the webpage: that should let us know when someone leaves a room

### ISSUE THREAD

**Issue by katjaq** [datalad-datasets/ohbm2020-posters#59]
Here's a simple repo showing how to embed many jitsies: https://github.com/r03ert0/manyjitsis

The code is basically:
```js
const domain = 'meet.jit.si';
  const {offsetWidth: width, offsetHeight: height} = document.querySelector("#jitsi");
  const options = {
    roomName: 'OpenForumMainRoom',
    width, height,
    parentNode: document.querySelector('#jitsi'),
    configOverwrite: {
      startWithAudioMuted: true
    }
  };
  let api = new JitsiMeetExternalAPI(domain, options);
```
which supposes there's a `div` with `id="jitsi"`.

**Comment by yarikoptic** [comment]
Attn @soichih

**Comment by soichih** [comment]
@katjaq I ended up doing this to detect for someone closing the window

```
function openJit(url, number) {
    wss.send(JSON.stringify({action: "jit", id: number}));
    let child = window.open(url, "jit"+number);
    let timer = setInterval(()=>{
        if(child.closed) {
            wss.send(JSON.stringify({action: "jitclose", id: number}));
            clearInterval(timer);
        }
    }, 1000);
}

```

This seems to work.. but I will try the `JitsiMeetExternalAPI`approach if it doesn't work still. Thank you for letting me know about this API!

**Comment by r03ert0** [comment]
awesome! we'll integrate that in [brainweb](https://brain-web.github.io/) as well. We started exploring using jitsi's data channel for allowing feedback (many other things could be done with it!). It would be wonderful if you guys were willing to keep pushing this with us after the OHBM meeting <3

**Comment by yarikoptic** [comment]
I think it was done by @soichih, so we are all set ! ;)

**Comment by r03ert0** [comment]
awesome!! \o/

**Comment by r03ert0** [comment]
Hello datafriends! we did a thing with @katjaq using your wss & data :)
We wanted to have an easier way of seeing who's connected and not. We made a screen with squares showing room occupancy (we also added the emoji feedback :p). Hope you'll like it! <3

https://brain-web.github.io/ohbm2020/index.html

**Comment by effigies** [comment]
That is really cool. Do we know anything about the number blocks? Could we split it by category to make it easier to narrow down which green squares to check out?

Or possibly add a search filter that will reduce the blocks to matching posters?

**Comment by r03ert0** [comment]
yes! we could change the background colour to reflect the category, or add a filter base on the keywords. I'll try to recycle brainweb's community graph viewer to make a network of similar posters (and whether they are active or not :D)

**Comment by yarikoptic** [comment]
abstracts.json already has bag of words for each abstract - might help to relate?
Coming from your ui to the room talks to soichis backend to announce count, right?
When you are ready (now?) we should add a list of alternative web UIs on top of our page.

**Comment by r03ert0** [comment]
now is good!

yes, I talk to @soichih's websocket, so that it doesn't matter which UI people use to connect, the counts will be the same :)

(checking abstracts.json, how cool!)

**Comment by yarikoptic** [comment]
BTW, forgot to mention open project which I hoped would be our gather town but it is one man effort AFAIK, but might give good ideas etc: https://github.com/capnmidnight/Calla

**Comment by yarikoptic** [comment]
I will reopen since we last discussion here...

**Comment by r03ert0** [comment]
how sweet!

**Comment by katjaq** [comment]
wow! this is a lovely gather space! thanks for sharing @yarikoptic hehe, then we can have you on gathertown, on jitsi, on zoom, and on this lovely calla gather space! :D mindblowing! ;)

**Comment by yarikoptic** [comment]
I have not tried calla for a while -- when I tried it was not yet ready, but if you try and see what/how to fixup -- we might end up indeed with our very own gather place!  Meanwhile -- added a quick reference to BrainWeb in 357b8c0c071cdbc58399f5b3b08aa9b8fc2598ae . Feel welcome to improve referencing/styling in the top of the page, whatever!
